mesonbuild/interpreter.py

```diff
@@ -2402,6 +2403,9 @@ def get_option_internal(self, optname):
 
         raise InterpreterException('Tried to access unknown option "%s".' % optname)
 
+    def get_all_user_options_internal(self, *args):
```

Comment:
Please name this function func_get_all_user_options like the other publicly exposed functions.